backend/app/assets/stylesheets/spree/backend/shared/_typography.scss

```diff
@@ -1,5 +1,10 @@
 // Base
 //--------------------------------------------------------------
+
+html {
+  font-size: $font-size-root;
```

Comment:
I would like that, but I think we should do this in a separate PR. Removing this would move us from `13px` to `16px` (maybe), which is a really significant change, which I'm sure will cause some issues. I'd rather we look at that separately.